internal/cmd/destinations.go

```diff
@@ -40,6 +41,7 @@ func newDestinationsListCmd(cli *CLI) *cobra.Command {
 				return err
 			}
 
+			logging.S.Debug("call server: list destinations")
```

Comment:
Any info about the current runtime we can add to these debug logs that might help give even more context? Not needed, just something I thought of.

Reply:
I have been struggling a bit attempting to error handle / log  'apiclient'/contexts, etc, since there is #1400 that needs to be addressed first